migrations/2_deploy_organization.js

```diff
@@ -0,0 +1,97 @@
+// Imports:
+const Avatar = artifacts.require("Avatar.sol");
+const Controller = artifacts.require("Controller.sol");
+
+const GlobalConstraintRegistrar = artifacts.require("GlobalConstraintRegistrar.sol");
+const TokenCapGC = artifacts.require("TokenCapGC.sol");
+
+const GenesisScheme = artifacts.require("GenesisScheme.sol");
+const SchemeRegistrar = artifacts.require("SchemeRegistrar.sol");
+const ContributionReward = artifacts.require("ContributionReward.sol");
+const UpgradeScheme = artifacts.require("UpgradeScheme.sol");
+const SimpleICO = artifacts.require("SimpleICO.sol");
+
+const AbsoluteVote = artifacts.require("AbsoluteVote.sol");
+
+// Instances:
+let AbsoluteVoteInst;
+let schemeRegistrarInst;
+let globalConstraintRegistrarInst;
+let upgradeSchemeInst;
+let ControllerInst;
+let AvatarInst;
+
+// DAOstack ORG parameters:
+const orgName = "Genesis";
+const tokenName = "Genes";
+const tokenSymbol = "GEN";
+const founders = [web3.eth.accounts[0]];
+const initRep = 10;
+const initRepInWei = [web3.toWei(initRep)];
+const initToken = 1000;
+const initTokenInWei = [web3.toWei(initToken)];
+let reputationAddress;
+let controllerAddress;
+
+// DAOstack parameters for universal schemes:
```

Comment:
We never decided on these params for DAOStack org and probably there are not the right params.
It might be usefull only for demo purpose..and if this is the case so it need to be mention that it is just for DEMO.

Reply:
The application is clear when it is running on a testnet.  Does arc-js need to do anything else?

Reply:
@dkent600 
> The application is clear when it is running on a testnet. 

Can you explain ^^^ ?
At least  add a comment that this script is just for demo and it got nothing related to the real DAOstack org .

I think the migration script should just deploy genesisscheme and all other universal schemes + voting machines. 
Then we might need another script which call genesisscheme to forge DAOstack org

Reply:
The header of the application looks like this

![image](https://user-images.githubusercontent.com/1821666/35069115-32dc83ae-fba7-11e7-81bd-67d54921ea4d.png)